assets/src/edit-story/components/inspector/design/designInspector.js

```diff
@@ -34,7 +34,8 @@ const Wrapper = styled.div`
 
 const TopPanels = styled.div`
   flex: 1 1;
-  overflow: auto;
+  overflow-y: overlay;
```

Comment:
This has been deprecated some time ago. If you still want it, please also keep the old `auto` above for cross-browser support. But I think since it's deprecated it might be best to avoid.